.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,6 +1,9 @@
+** NOTE: Do NOT add any Jira links to this PR description or commits.
+This is a public repository.**
```

Comment:
> This does not markdown.
 
Lord.

This does now markdown, although it took me [two][1] [tries][2].

[1]: https://github.com/fastly/fastly_nsq/pull/35/commits/d2c0524f18520670cd75afbabc46a85033e513e5
[2]: https://github.com/fastly/fastly_nsq/pull/35/commits/9536dc41fb4189a625b41862b0b91900ace33e7d